superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx

```diff
@@ -43,6 +43,9 @@ interface DatabaseModalProps {
 }
 
 const DEFAULT_TAB_KEY = '1';
+const EXPOSE_SQLLAB_FORM_HEIGHT = '260px';
```

Comment:
can you bump this up to 270px? It's cutting some off. This approach is admittedly brittle. Maybe we can try to find a way of capturing the current height of the children and adding it up and passing it to emotion for styling in the next iteration of this.